Source/inv.cpp

```diff
@@ -1563,7 +1570,7 @@ void AutoGetItem(int pnum, int ii)
 			if (w == 1 && h == 1) {
 				idx = plr[pnum].HoldItem.IDidx;
 				if (plr[pnum].HoldItem._iStatFlag && AllItemsList[idx].iUsable) {
-					for (i = 0; i < 8 && !done; i++) {
+					for (i = 0; i < MAXBELTITEMS && !done; i++) {

```

Comment:
this should go to devilution in a separate PR

Reply:
https://github.com/diasurgical/devilution/pull/2071